doc/contributing/releases.md

```diff
@@ -1018,6 +1018,19 @@ This script will use the promoted builds and changelog to generate the post. Run
   "Set as the latest release".
 * Click on the "Publish release" button.
 
+#### (Temporary) Update the Cloudflare R2 Worker Symlinks
+
+We have adopted Cloudflare R2 and Cloudflare Workers for our
+Release Assets (see <https://github.com/nodejs/build/issues/3461>).
+
+For now, after a release is done, the symlinks need to be updated manually.
+This can be done by simply
+[triggering this GitHub Worfklow](https://github.com/nodejs/release-cloudflare-worker/actions/workflows/update-links.yml).
```

Comment:
```suggestion
[triggering this GitHub Worfklow](https://github.com/nodejs/release-cloudflare-worker/actions/workflows/update-links.yml), and then [deploying](https://github.com/nodejs/release-cloudflare-worker/actions/workflows/deploy.yml) the update
```

Reply:
Is the manual deployment really needed? Doesn't the sync already deploy it?

Reply:
we can add that to the sync workflow, but it currently only opens a PR